cpg_flow_test/jobs/build_pyramid.py

```diff
@@ -50,7 +51,7 @@ def build_pyramid(
         sg_jobs.append(job)
 
     # Merge the no evens lists for all sequencing groups into a single file
-    job = b.new_job(name=title)
+    job = b.new_job(name=title, attributes=job_attrs | {'tool': 'cat'})
```

Comment:
tool cat? 😂

Reply:
Yep 😸 

The command for this job is just `job.command(f'cat {inputs} >> {job.pyramid}')`. But really this is just an example of unioning in some more attributes entries.